patroni/utils.py

```diff
@@ -1211,3 +1211,36 @@ def get_major_version(bin_dir: Optional[str] = None, bin_name: str = 'postgres')
     if TYPE_CHECKING:  # pragma: no cover
         assert version is not None
     return '.'.join([version.group(1), version.group(3)]) if int(version.group(1)) < 10 else version.group(1)
+
+
+def get_postgres_version(bin_dir: Optional[str] = None, bin_name: str = 'postgres') -> str:
+    """Get full PostgreSQL version.
+
+    Like func:`get_major_version` but extended with minor version.
```

Comment:
IMO `get_major_version()` should simply call this function and remove last `r'\.\d+'` from the returned value.